Shared/ThresholdKeyView.swift

```diff
@@ -364,35 +421,48 @@ alertContent = "There are \(totalShares) available shares. \(key_details.require
                             LoaderView()
                         }
                         Button(action: {
-                            // TODO: allow users to input password in a popup.
-                            // TODO: add loader as well, API call could take a >3 seconds
-                            let question = "what's your password?"
-                            let answer = randomPassword()
-                            Task {
-                                do {
-                                    showSpinner = SpinnerLocation.add_password_btn
-                                    let share = try await SecurityQuestionModule.generate_new_share(threshold_key: threshold_key, questions: question, answer: answer)
-                                    print(share.share_store, share.hex)
+                        Task {
+                            showInputPasswordAlert.toggle()
+                        }
+                        }){
+                            Text("")
+                        }.alert("Enter Password", isPresented: $showInputPasswordAlert) {
+                            SecureField("Password", text: $password)
```

Comment:
Should there not be post processing on inputted text to ensure they are using a password that is sufficiently difficult to be considered secure? i.e 12 character minimum, use of special characters, numbers and capitalization?

Reply:
In our example application, I don't think the difficulty of the password matters as much as the user's ability to quickly set and confirm the password, so I'm not placing any conditions on it.

Reply:
And if the user uses an actual rather than test account here? This is possible. Then we have allowed them to set an insecure password for the security share.

Reply:
insecure password is okay here